Enhanced `GenericMap.draw_contours()` to be able to plot on a different WCSAxes (#5239)
sunpy/sunpy: changelog/5239.bugfix.rst

> Fixed :meth:`sunpy.map.GenericMap.draw_contours` so that the contours can be plotted on axes that correspond to a different map.

Reviewer: ```suggestion
Fixed :meth:`sunpy.map.GenericMap.draw_contours` so that the contours from a map can be plotted on axes with a different coordinate system.
```
